python/cudf/cudf/tests/test_string.py

```diff
@@ -874,6 +874,27 @@ def test_string_contains(ps_gs, pat, regex, flags, flags_raise, na, na_raise):
         assert_eq(expect, got)
 
 
+@pytest.mark.parametrize(
+    "ptn,esc,expect",
```

Comment:
For some reason I still can't comprehend `ptn` as an abbreviation for `pattern`. We should at least use `pat` to match the tests above, but I prefer the full name.
```suggestion
    "pattern,esc,expect",
```

Reply:
I'd prefer to use `pat` to match `contains` parameter.

Reply:
Sorry, wrote my suggestion incorrectly (I meant to prioritize consistency with above tests). Go for `pat` here.